apps/spruce/cypress/integration/image/event_log.ts

```diff
@@ -0,0 +1,12 @@
+describe("event log page", () => {
```

Comment:
Are there any more tests we can have here? Could we test searching for a nonexistent item?